Failed build of docs in CI

```
#!/bin/bash -eo pipefail
cd docs
make html
npm install && touch node_modules
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.1.2 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN jupyterhub-docs-build@0.8.0 No repository field.

added 216 packages from 514 contributors and audited 325 packages in 4.188s

14 packages are looking for funding
  run `npm fund` for details

found 2 low severity vulnerabilities
  run `npm audit fix` to fix them, or `npm audit` for details
npm run rest-api

> jupyterhub-docs-build@0.8.0 rest-api /home/circleci/project/docs
> bootprint openapi ./rest-api.yml source/_static/rest-api

[
  'source/_static/rest-api/index.html',
  'source/_static/rest-api/main.css',
  'source/_static/rest-api/main.css.map'
]
sphinx-build -b html -d build/doctrees  "-W" source build/html
Running Sphinx v2.4.4
Adding copy buttons to code blocks...
making output directory... done
/home/circleci/.local/lib/python3.6/site-packages/sphinx/util/compat.py:32: RemovedInSphinx30Warning: The config variable "source_parsers" is deprecated. Please update your extension for the parser and remove the setting.
  RemovedInSphinx30Warning)
/home/circleci/.local/lib/python3.6/site-packages/sphinx/util/compat.py:36: RemovedInSphinx30Warning: app.add_source_parser() does not support suffix argument. Use app.add_source_suffix() instead.
  app.add_source_parser(suffix, parser)

Theme error:
no theme named 'pandas_sphinx_theme' found (missing theme.conf?)
Makefile:64: recipe for target 'html' failed
make: *** [html] Error 2
```

FYI @choldgraf @betatim

--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -3,7 +3,7 @@
 -r ../requirements.txt
 alabaster_jupyterhub
 autodoc-traits
-https://github.com/pandas-dev/pandas-sphinx-theme/archive/ade576c92cfe46a372b6783fb83f45c44fc67976.zip
+pydata-sphinx-theme
 recommonmark>=0.6
 sphinx-copybutton
 sphinx-jsonschema

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -74,7 +74,7 @@ source_suffix = ['.rst', '.md']
 # -- Options for HTML output ----------------------------------------------
 
 # The theme to use for HTML and HTML Help pages.
-html_theme = 'pandas_sphinx_theme'
+html_theme = 'pydata_sphinx_theme'
 
 html_logo = '_static/images/logo/logo.png'
 html_favicon = '_static/images/logo/favicon.ico'

